kratos/utilities/element_size_calculator.cpp

```diff
@@ -116,6 +116,33 @@ double ElementSizeCalculator<2,3>::MinimumElementSizeDerivative(
     KRATOS_CATCH("");
 }
 
+// Triangle2D6 version.
+template<>
+double ElementSizeCalculator<2,6>::MinimumElementSize(const Geometry<Node<3> >& rGeometry)
+{
+
+    const double minimum_element_size = ElementSizeCalculator<2,3>::MinimumElementSize(rGeometry);
+
+
+    return minimum_element_size;
+}
+// Triangle2D6 version.
+template<>
```

Comment:
```suggestion
}

// Triangle2D6 version.
template<>
```
Very minor to ease going through the code.